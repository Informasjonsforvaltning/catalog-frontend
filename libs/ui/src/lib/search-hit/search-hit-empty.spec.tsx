import React from 'react';
import { render } from '@testing-library/react';
import { SearchHit } from './';

describe('SearchHit', () => {
  it('should render SearchHit with undefined input successfully', () => {
    const { baseElement } = render(
      <SearchHit
        searchHit={{}}
        catalogId={''}
      />,
    );
    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
