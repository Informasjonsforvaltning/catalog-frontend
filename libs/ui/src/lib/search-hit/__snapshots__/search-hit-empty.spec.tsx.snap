// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchHit should render SearchHit with undefined input successfully 1`] = `
<body>
  <div>
    <div
      class="container"
    >
      <div
        class="rowSpaceBetween"
      >
        <div
          class="titleRow"
        >
          <a
            href=""
          >
            <h2
              class="title"
            >
              Uten navn
            </h2>
          </a>
        </div>
      </div>
    </div>
  </div>
</body>
`;
